custom_components/tuya_local/devices/smart_weather_station.yaml

```diff
@@ -0,0 +1,93 @@
+name: Smart Weather Station
```

Comment:
HA naming convention: Only capitalise the first word of a name (unless a trademark or abbreviation that requires it)

Reply:
This is how device introduces itself in tuya app, but I am ok with changing it.